<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-38MES30" failures="6" tests="12" name="AutoQA" time="12.975" errors="0" timestamp="2019-09-09T09:18:48 EEST">
  <testcase classname="io.swagger.petstore.OrderTest" name="orderTest" time="4.971"/>
  <testcase classname="io.swagger.petstore.PetStoreTest" name="addPetWithInvalidData" time="0.578">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	StatusCode isn&amp;apos;t equals expected [200] but found [500]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
StatusCode isn't equals expected [200] but found [500]
at asertions.BaseAssertions.baseAssert(BaseAssertions.java:20)
at business.PetBL.addPetWithInvalidDataTest(PetBL.java:71)
at io.swagger.petstore.PetStoreTest.addPetWithInvalidData(PetStoreTest.java:62)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- addPetWithInvalidData -->
  <testcase classname="io.swagger.petstore.PetStoreTest" name="deleteNotPresentPet" time="0.422"/>
  <testcase classname="io.swagger.petstore.PetStoreTest" name="getInvalidPet" time="0.36"/>
  <testcase classname="io.swagger.petstore.PetStoreTest" name="getPetByStatus" time="0.522"/>
  <testcase classname="io.swagger.petstore.PetStoreTest" name="petTest" time="2.533"/>
  <testcase classname="io.swagger.petstore.PetStoreTest" name="uploadImage" time="0.909"/>
  <testcase classname="io.swagger.petstore.UserTest" name="TestLoginAndLogoutUser" time="0.375">
    <failure type="com.fasterxml.jackson.databind.exc.MismatchedInputException" message="No content to map due to end-of-input
 at [Source: (String)&amp;quot;&amp;quot;; line: 1, column: 0]">
      <![CDATA[com.fasterxml.jackson.databind.exc.MismatchedInputException: No content to map due to end-of-input
 at [Source: (String)""; line: 1, column: 0]
at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4145)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4000)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3042)
at com.fasterxml.jackson.databind.ObjectMapper$readValue$2.call(Unknown Source)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.groovy:46)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize$0.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:221)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$1.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:58)
at io.restassured.internal.mapping.ObjectMapping$deserialize$0.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:224)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at business.UserBL.addNewUserTest(UserBL.java:56)
at io.swagger.petstore.UserTest.TestLoginAndLogoutUser(UserTest.java:41)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- TestLoginAndLogoutUser -->
  <testcase classname="io.swagger.petstore.UserTest" name="UpdateUserTest" time="0.375">
    <failure type="com.fasterxml.jackson.databind.exc.MismatchedInputException" message="No content to map due to end-of-input
 at [Source: (String)&amp;quot;&amp;quot;; line: 1, column: 0]">
      <![CDATA[com.fasterxml.jackson.databind.exc.MismatchedInputException: No content to map due to end-of-input
 at [Source: (String)""; line: 1, column: 0]
at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4145)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4000)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3042)
at com.fasterxml.jackson.databind.ObjectMapper$readValue$2.call(Unknown Source)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.groovy:46)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize$0.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:221)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$1.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:58)
at io.restassured.internal.mapping.ObjectMapping$deserialize$0.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:224)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at business.UserBL.addNewUserTest(UserBL.java:56)
at io.swagger.petstore.UserTest.UpdateUserTest(UserTest.java:50)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- UpdateUserTest -->
  <testcase classname="io.swagger.petstore.UserTest" name="addArrayOfUserTest" time="0.32">
    <failure type="com.fasterxml.jackson.databind.exc.MismatchedInputException" message="No content to map due to end-of-input
 at [Source: (String)&amp;quot;&amp;quot;; line: 1, column: 0]">
      <![CDATA[com.fasterxml.jackson.databind.exc.MismatchedInputException: No content to map due to end-of-input
 at [Source: (String)""; line: 1, column: 0]
at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4145)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4000)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3042)
at com.fasterxml.jackson.databind.ObjectMapper$readValue$2.call(Unknown Source)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.groovy:46)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize$0.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:221)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$1.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:58)
at io.restassured.internal.mapping.ObjectMapping$deserialize$0.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:224)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at business.UserBL.addArrayOfUsersTest(UserBL.java:70)
at io.swagger.petstore.UserTest.addArrayOfUserTest(UserTest.java:31)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- addArrayOfUserTest -->
  <testcase classname="io.swagger.petstore.UserTest" name="addListOfUserstest" time="0.329">
    <failure type="com.fasterxml.jackson.databind.exc.MismatchedInputException" message="No content to map due to end-of-input
 at [Source: (String)&amp;quot;&amp;quot;; line: 1, column: 0]">
      <![CDATA[com.fasterxml.jackson.databind.exc.MismatchedInputException: No content to map due to end-of-input
 at [Source: (String)""; line: 1, column: 0]
at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4145)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4000)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3042)
at com.fasterxml.jackson.databind.ObjectMapper$readValue$2.call(Unknown Source)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.groovy:46)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize$0.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:221)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$1.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:58)
at io.restassured.internal.mapping.ObjectMapping$deserialize$0.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:224)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at business.UserBL.addListOfUsersTest(UserBL.java:81)
at io.swagger.petstore.UserTest.addListOfUserstest(UserTest.java:36)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- addListOfUserstest -->
  <testcase classname="io.swagger.petstore.UserTest" name="firstTest" time="0.514">
    <failure type="com.fasterxml.jackson.databind.exc.MismatchedInputException" message="No content to map due to end-of-input
 at [Source: (String)&amp;quot;&amp;quot;; line: 1, column: 0]">
      <![CDATA[com.fasterxml.jackson.databind.exc.MismatchedInputException: No content to map due to end-of-input
 at [Source: (String)""; line: 1, column: 0]
at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4145)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4000)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3042)
at com.fasterxml.jackson.databind.ObjectMapper$readValue$2.call(Unknown Source)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.groovy:46)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize$0.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:221)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$1.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:58)
at io.restassured.internal.mapping.ObjectMapping$deserialize$0.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:224)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at business.UserBL.addNewUserTest(UserBL.java:56)
at io.swagger.petstore.UserTest.firstTest(UserTest.java:25)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- firstTest -->
</testsuite> <!-- AutoQA -->
